components/compliance-service/reporting/relaxting/assets.go

```diff
@@ -441,3 +441,37 @@ func (backend ES2Backend) getUnCollectedAssets(ctx context.Context, from int32,
 	filtQuery.Must(getStartTimeAndEndTimeRangeForAsset(filters))
 	return backend.getAssetsList(ctx, from, size, filtQuery)
 }
+func (backend ES2Backend)GetAsset(ctx context.Context, filters map[string][]string , size int32 , from int32 , asset_type string) ([]*reportingapi.Assets, error) {
+    boolquery := backend.getFiltersQueryForAssetFilters(filters)
+    assets := make([]*reportingapi.Assets, 0)
+    if asset_type == "unreachable" {
+        unreachable,err := backend.getUnReachableAssets(ctx , from , size , boolquery , 10)
+        if err != nil {
+            logrus.Errorf("Unable to get the unreachable assets %v" , err)
```

Comment:
```suggestion
            logrus.Errorf("Unable to get the unreachable assets: %v" , err)
```
separate comment and error by `:`